<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="SAMEERS-T460" name="com.incontact.jmeter.wrapper.JMeterConfigurator" tests="1" failures="0" timestamp="25 Apr 2017 05:24:54 GMT" time="0.156" errors="1">
  <testcase name="createJmeterJmxFile" time="0.156" classname="com.incontact.jmeter.wrapper.JMeterConfigurator">
    <error type="org.apache.jorphan.util.JMeterError" message="JMeter requires the checksum of saveservice properties file to continue">
      <![CDATA[org.apache.jorphan.util.JMeterError: JMeter requires the checksum of saveservice properties file to continue
	at org.apache.jmeter.save.SaveService.initProps(SaveService.java:212)
	at org.apache.jmeter.save.SaveService.<clinit>(SaveService.java:162)
	at com.incontact.jmeter.wrapper.JMeterConfigurator.createJmeterJmxFile(JMeterConfigurator.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:326)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: java.io.FileNotFoundException: null\bin\saveservice.properties (The system cannot find the path specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileReader.<init>(Unknown Source)
	at org.apache.jmeter.save.SaveService.getChecksumForPropertiesFile(SaveService.java:196)
	at org.apache.jmeter.save.SaveService.initProps(SaveService.java:209)
	... 21 more
]]>
    </error>
  </testcase> <!-- createJmeterJmxFile -->
</testsuite> <!-- com.incontact.jmeter.wrapper.JMeterConfigurator -->
